contessa/executor.py

```diff
@@ -67,14 +67,26 @@ def compose_where_time_filter(self, rule):
         only data that were updated/created/confirmed in last 30 days.
         :return: str, WHERE `time_filter` filter statement
         """
-        time_filter_column = rule.time_filter
-        if time_filter_column:
-            past = (datetime.now() - timedelta(days=30)).strftime(
-                "%Y-%m-%d %H:%M:%S UTC"
-            )
-            return f"""{time_filter_column} >= '{past}'::timestamp"""
-        else:
-            return ""
+        days = 30
+        filters = []
+        result = []
+        if rule.time_filter:
```

Comment:
I know we do not have a doc, but for this feature it would be nice to it. Could you start some separate folder docs and just document there how this filters works with examples ? Then I'll try to add more stuff to it and then we can host it somewhere. 
So, basically just to start having docs. thanks

Reply:
I'll add format description to the comment of method. Other stuff could be done in #1